cmd/substreams/init.go

```diff
@@ -210,7 +210,7 @@ func renderProjectFilesIn(project templates.Project, absoluteProjectDir string)
 	}
 
 	for relativeFile, content := range files {
-		file := path.Join(absoluteProjectDir, strings.ReplaceAll(relativeFile, "/", string(os.PathSeparator)))
+		file := path.Join(absoluteProjectDir, relativeFile)
```

Comment:
I'm surprised of this change, here we use `os.WriteFile` so I'm expecting that `file` should use OS path separator. So I would see the change more as:

```suggestion
		file := filepath.Join(absoluteProjectDir, strings.ReplaceAll(relativeFile, "/", string(os.PathSeparator)))
```

Reply:
Ok `absoluteProjectDir` is also problematic since it's wrongly built, need to change this line [absoluteProjectDir](https://github.com/streamingfast/substreams/blob/b9cef0fef35e45c5f3cd0f7e976915f09aad6ded/cmd/substreams/init.go#L75) to become `	absoluteProjectDir := filepath.Join(absoluteWorkingDir, projectName)
`

Reply:
After the suggested change [here](https://github.com/streamingfast/substreams/blob/b9cef0fef35e45c5f3cd0f7e976915f09aad6ded/cmd/substreams/init.go#L75) init is failing

`Error: render Ethereum Ethereum Mainnet project: removing directory .: RemoveAll .: invalid argument and write file: open F:\Projects\test_init\new_code\substreams.yaml: The system cannot find the path specified.`

Reply:
Ahhh [this](https://github.com/streamingfast/substreams/blob/b9cef0fef35e45c5f3cd0f7e976915f09aad6ded/cmd/substreams/init.go#L186) need to change as well to use filepath and not path.